generator/tag_admixer.hpp

```diff
@@ -147,66 +147,85 @@ class TagAdmixer
 class TagReplacer
 {
 public:
+  using Replacements = std::map<OsmElement::Tag, std::vector<OsmElement::Tag>>;
+
   TagReplacer() = default;
 
   explicit TagReplacer(std::string const & filePath)
   {
     std::ifstream stream(filePath);
 
-    OsmElement::Tag tag;
-    std::vector<std::string> values;
     std::string line;
     while (std::getline(stream, line))
     {
-      if (line.empty())
+      strings::Trim(line);
+      if (line.empty() || line[0] == '#')
         continue;
 
-      strings::SimpleTokenizer iter(line, " \t=,:");
-      if (!iter)
-        continue;
-      tag.m_key = *iter;
-      ++iter;
-      if (!iter)
-        continue;
-      tag.m_value = *iter;
+      auto keyIt = line.find("=");
```

Comment:
nit: s/keyIt/keyPos/
find does not return an iterator

Reply:
done